dev/bin/create-stable-pr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+COMMIT_HASH=${1}
+COMMIT_DESCRIPTION=$(git log --pretty=format:"%h %s" -1 $COMMIT_HASH)
+
+echo "DEV COMMIT: $COMMIT_HASH"
+echo "$COMMIT_DESCRIPTION"
+
+export MANAGER_NEW_TAG=$(git show $COMMIT_HASH:kustomize/overlays/prod/kustomization.yaml | yq .images[0].newTag)
```

Comment:
where does `yq` comes from ? I think it is not installed default on Fedora

Maybe add some doc to install it as a dependency (like `jq`)

Reply:
@radtriste added GitHub link. I wouldn't put it in the doc as it's needed only for this script

Reply:
IMO as long as we use a tool in one of the dev scripts, then we should document in the dev README file (and maybe put `optional` mention)
It could be that the tool would be reuse later in some other script(s), that would avoid to have to think about adding it afterwards. It should be mentioned the first time we introduce it ;)